### ISSUE TITLE
MoreSlots Mod use compatibility

### ISSUE THREAD

**Issue by whitewave** [RandyKnapp/Auga#62]
Before using MoreSlots  by Fang86 mod

![MoreSlot mod_not_use](https://user-images.githubusercontent.com/6347659/134282006-e82f54fc-d171-4c3c-99ad-b1acee465383.png)

After using MoreSlots by Fang86 mod
![MoreSlot_mod_use](https://user-images.githubusercontent.com/6347659/134282099-d065e112-1fa9-4349-b473-ed94990f2ef0.png)

1. Inventory scroll created
2. meaningless empty space
3. Exceeds box inventory screen resolution

How can I correct the above 3 items?

**Comment by aeger** [comment]
Yes please! I'm trying to switch from Valheim Plus to OdinQOL and losing the 2 rows of player inventory is really messing me up.

**Comment by Caaruzo** [comment]
Same for me with Azumat ExtendedInventory.
Would love to see scaling ui for more inventory rows instead scroll. So there can be added 2 -3 without scrollbar.
Thx much
